src/medkit/medkit.tsx

```diff
@@ -120,59 +146,39 @@ export const Medkit: React.FC<MedkitParams> = ({scenario, budget, medkitId}: Med
-                                        <th scope="col">Select</th>
-                                    </tr>
-                                </thead>
-                                <tbody>
-                                    {itemsPicked.every((el) => !el) ? (
-                                        <tr>
-                                            <th scope={'row'} colSpan={4}>
-                                                Your kit has not been packed yet.
-                                            </th>
-                                        </tr>
-                                    ) : (
-                                        <>
-                                            {DATA.map((el, idx) => {
-                                                return (
-                                                    <React.Fragment key={idx}>
-                                                        {itemsPicked[idx] && (
-                                                            <tr>
-                                                                <th scope="row">{el.category}</th>
-                                                                <td>{el.item}</td>
-                                                                <td>{el.points}</td>
-                                                                <td>
-                                                                    <button type="button"
-                                                                        className="btn btn-light"
-                                                                        id={String(idx)}
-                                                                        onClick={handlePickItem}>
-                                                                        Remove
-                                                                    </button>
-                                                                </td>
-                                                            </tr>
-                                                        )}
-                                                    </React.Fragment>
-                                                );
-                                            })}
-                                        </>
-                                    )}
-                                </tbody>
-                                <tfoot>
-                                    <tr>
-                                        <td></td>
-                                        <td>{totalItemsPicked} Items</td>
-                                        <td>{totalItemsScore} Points</td>
-                                        <td></td>
-                                    </tr>
-                                </tfoot>
-                            </table>
+                            <div className={'row bg-dark p-3 mx-0 text-light'}>
+                                <div className="col-5">
```

Comment:
The decision to pull the items off the right-hand column was the correct one. It's much easier to see what's going on. That said, the right-hand column does need some layout attention, with some elements centered and others not.

Also - on smaller resolutions, having this float to the top would create a workable interface. (That's simply a matter of adding bootstrap `order` indicators. This is what I do in DESIGN:

```
<div class="row">
    <div class="col-lg-3 order-lg-2">Right-hand column, which floats to the top at larger rez</div>
    <div class="col-lg-9 order-lg-1">Left-hand column, which floats to the bottom at larger rez</div>
</div>
```